tfhe/src/integer/gpu/server_key/radix/ilog2.rs

```diff
@@ -582,14 +571,14 @@ impl CudaServerKey {
         let mut message_blocks_slice = message_blocks
             .0
             .d_vec
-            .as_mut_slice(0..lwe_size * counter_num_blocks, streams.gpu_indexes[0].0)
+            .as_mut_slice(0..lwe_size * counter_num_blocks, 0)
             .unwrap();
         let result_slice = result
             .as_mut()
             .d_blocks
             .0
             .d_vec
-            .as_slice(0..lwe_size * counter_num_blocks, streams.gpu_indexes[0].0)
+            .as_slice(0..lwe_size * counter_num_blocks, 0)
```

Comment:
the second argument in as_slice  it means to use the gpu that is in the index 0 of the array of available GPUs, right?

Reply:
Yes :+1:

Reply:
There was a confusion actually and this PR fixes it.